ocs_ci/deployment/ocp.py

```diff
@@ -190,5 +191,8 @@ def destroy(self, log_level="DEBUG"):
             logger.info(f"Destroying cluster defined in {self.cluster_path}")
             run_cmd(destroy_cmd)
 
+        except CommandFailed:
```

Comment:
Is the intention to "ignore" all other types of Exceptions and just log them? If we want to raise those as well should we just add the `raise` to the `except Exception` block (line 197)?

Reply:
Initially, the behaviour was exactly like that - ignore all exceptions and just log them, so I don't want to change it too much (as https://github.com/red-hat-storage/ocs-ci/issues/1352 was the only issue we faced till now).

I'm aware of `FileNotFoundError` exception raised from line 186, when `metadata.json` file does not exist and this exception is expected and we want to catch it without failing the destroy process (this case usually means, that the cluster wasn't installed at all, so there is nothing to destroy - but we want to clean up other auxiliary resources - e.g. jenkins slave).

I agree, that it might be safer to explicitly catch only that *known/expected* exception and re-raise any other - but this proposed change is closer to the previous behaviour (but I don't have strong opinion about this, so if you think that it would be better to change it, I can do that).

Reply:
I agree with your take on it. If this is the only issue we have encountered let's simply address that without changing the logic here drastically.